dbtwiz/main.py

```diff
@@ -5,6 +5,7 @@
 
 from dbtwiz import admin
 from dbtwiz import commands
+from dbtwiz import generate
```

Comment:
Could we sort the available commands listed when using `--help`?
I think the commands should be listed in order of most commonly used by business users, so possibly `generate` and `sqlfix` should be somewhat high up in the list.
Alternatively, the list should be ordered alphabetically.

Are all the options in use? E.g., when would someone opt to use the dbtwiz commands `build` or `test` rather than `dbt build` or `dbt test`?
And `freshness` and `manifest` is probably not something a business user would do, so maybe move under `admin`?

![image](https://github.com/user-attachments/assets/aa62f032-8f77-47e0-8459-dde897b5d3df)

Reply:
Good ideas, but let's move them into separate issues.

Reply:
The `model` command is just a placeholder for now, the idea is to use it for inspecting a given model to show as much info about it as possible without having to dig through multiple files.

The `build` command uses `fzf` to quickly select which model(s) to build interactively, if you don't already have the full name in the paste buffer.

Reply:
Created issue: https://github.com/amedia/dbtwiz/issues/30